tests/unit/plugins/module_utils/test_syslog_server.py

```diff
@@ -23,6 +23,9 @@
     reason=f"requires python{MIN_PYTHON_VERSION[0]}.{MIN_PYTHON_VERSION[1]} or higher",
 )
 
+PROTOCOL_TCP = "SYSLOG_PROTOCOL_TCP"
+ANSIBLE_TCP = "tcp"
```

Comment:
Variable names like `HYPERCORE_PROTOCOL_TCP` and `ANSIBLE_PROTOCOL_TCP` would be more explicit.